internal/cmd/argus/grafana/describe/describe.go

```diff
@@ -0,0 +1,129 @@
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Shows details of the Grafana configuration of an Argus instance",
+		Long:  "Shows details of the Grafana configuration of an Argus instance.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of the Grafana configuration of an Argus instance with ID "xxx"`,
+				"$ stackit argus credentials describe --instance-id xxx"),
+			examples.NewExample(
+				`Get details of of the Grafana configuration of an Argus instance with ID "xxx" in a table format`,
+				"$ stackit argus credentials describe --instance-id xxx --output-format pretty"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			grafanaConfigsReq := buildGrafanaConfigRequest(ctx, model, apiClient)
+			grafanaConfigsResp, err := grafanaConfigsReq.Execute()
+			if err != nil {
+				return fmt.Errorf("get Grafana configs: %w", err)
+			}
+			instanceReq := buildInstanceRequest(ctx, model, apiClient)
+			instanceResp, err := instanceReq.Execute()
+			if err != nil {
+				return fmt.Errorf("get instance: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, grafanaConfigsResp, instanceResp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		InstanceId:      flags.FlagToStringValue(cmd, instanceIdFlag),
+	}, nil
+}
+
+func buildGrafanaConfigRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiGetGrafanaConfigsRequest {
+	req := apiClient.GetGrafanaConfigs(ctx, model.InstanceId, model.ProjectId)
+	return req
+}
+
+func buildInstanceRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiGetInstanceRequest {
+	req := apiClient.GetInstance(ctx, model.InstanceId, model.ProjectId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, grafanaConfigs *argus.GrafanaConfigs, instance *argus.GetInstanceResponse) error {
+	switch outputFormat {
+	case globalflags.PrettyOutputFormat:
+
+		table := tables.NewTable()
+		table.AddRow("GRAFANA DASHBOARD", *instance.Instance.GrafanaUrl)
+		table.AddSeparator()
+		table.AddRow("PUBLIC READ ACCESS", *grafanaConfigs.PublicReadAccess)
+		table.AddSeparator()
+		table.AddRow("SINGLE SIGN-ON", *grafanaConfigs.UseStackitSso)
```

Comment:
In the Grafana tab, the portal shows the initial Admin login to Grafana. We should probably do something similar in the CLI, right? 

In the instance describe this is not shown, so maybe here is the place if we are going to do it.

Reply:
Great point, added those